opal/mca/accelerator/cuda/accelerator_cuda.c

```diff
@@ -538,3 +542,30 @@ static int accelerator_cuda_device_can_access_peer(int *access, int dev1, int de
     }
     return 0;
 }
+
+/*
+ * Get the buffer ID from the memory and store it in the registration.
```

Comment:
This comment no longer matches the code (you don't store it in the registration in this code).